### ISSUE TITLE
Narrative schema documentation

### ISSUE THREAD

**Issue by pwadsworth** [gicentre/litvis#21]
Interested in the possibilities of narrative schemas with litvis but found insuficient documentation to get started. Can you link to additional resources?

**Comment by jwoLondon** [comment]
Yes, I will look at providing a more comprehensive documentation on narrative schemas.

(I assume you have already seen [this documentation](https://github.com/gicentre/litvis/tree/master/documents#narrative-schemas) and [these examples](https://github.com/gicentre/litvis/tree/master/examples)).
